projects/documentation/src/getting-started.ts

```diff
@@ -10,5 +10,4 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import '@spectrum-web-components/theme/theme-lightest.js';
 import '@spectrum-web-components/theme/scale-medium.js';
```

Comment:
I'm not 100% sure how this is used anymore, but this should _likely_ be `large` and `dark` as seen in `getting-started.md` or removed all together. 🤔